.github/workflows/ci-e2e.yml

```diff
@@ -168,7 +166,8 @@ jobs:
                   build-id: ${{ needs.container.outputs.build-id }}
                   tags: ${{ needs.container.outputs.tag }}
 
-            - name: Write .env # This step intentionally has no if, so that GH always considers the action as having run
```

Comment:
I've tested this, we don't need it, there are other required actions that are skipping every single step. We need the `action` to not have any `if`, but it's fine if all of the steps are skipped